bin/recog_cleanup

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
+
+$:.unshift(File.expand_path(File.join(File.dirname(__FILE__), "..", "lib")))
+require 'optparse'
+require 'ostruct'
+require 'recog'
+
+# Cleanup trailing whitespace an
+Dir[ File.expand_path(File.join(File.dirname(__FILE__), "..", "xml")) + "/*.xml" ].each do |f|
+    data = File.read(f).
+        gsub(/\s+$/, '').                           # Trailing whitespace and empty lines
```

Comment:
This is removing whitespace inside of comments. I think these should be left for readability. In the `sed` command I checked for a single newline after `</fingerprint>` so that we could safely replace without cumulative line additions.

Reply:
This only seemed to impact a couple comments was much less complex than the xmllint+sed line. For my dev case at least (running this on Windows), getting a working cli xmllint is a pain, not to mention lack of support for parallel, etc.

Maybe we can switch to a REXML or Nokogiri-based formatted instead?

Reply:
+++ For platform independent commands. Might be able to tweak the regex in the following command, I'll take a look later today.